packages/number-input/src/Input/Input.tsx

```diff
@@ -162,23 +163,23 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
       }
     };
 
-    const handleOnFocus = () => {
+    const handleFocusContainer = () => {
       isFocusedRef.current = true;
       handleSetErrorTransition();
     };
 
-    const handleOnBlur = () => {
+    const handleBlurContainer = () => {
```

Comment:
made these more explicit to differentiate with `onBlur` prop passed into the api

Reply:
nitpick: should these be `handleContainerBlur/Focus` instead?